include/ExtrapolatedSmoother/ExtrapolatedSmootherGive/extrapolatedSmootherGive.h

```diff
@@ -17,11 +17,13 @@ class ExtrapolatedSmootherGive : public ExtrapolatedSmoother
 
     ~ExtrapolatedSmootherGive() override;
 
-    void extrapolatedSmoothing(Vector<double>& x, const Vector<double>& rhs, Vector<double>& temp) override;
+    void extrapolatedSmoothing(Vector<double> x,
+                               const Vector<double> rhs,
+                               Vector<double> temp) override;
```

Comment:
I am surprised at the change of indenting given that the line is now shorter. Is this the indenting recommended by clang-formatting?

Maybe there is a step missing from the CI to enforce the clang-formatting?